draft-ietf-rats-architecture.md

```diff
@@ -354,72 +354,65 @@ while other slots cannot. But other slots can communicate with the main
-'------------------------------------------------------------------'
+              .-----------------------------.
+              |           Verifier          |
+              '-----------------------------'
+                             ^
+                             | Evidence of
+                             | Composite Device
+                             |
+.----------------------------|----------------------------------.
+| .--------------------------|--------.      .------------.     |
+| |                .----------------. |<-----| Attester B |     |
+| |  Collect       | Attesting      | |      '------------'-.   |
+| |  Claims .----->| Environment(s) | |<-------| Attester C |   |
+| |         |      '----------------' |        '------------'-. |
+| | .----------------.                |<---------| ...        | |
+| | | Target         |                | Evidence '------------' |
+| | | Environment(s) |                | of Attesters            |
+| | '----------------'   Attester A   | (via Internal Links or  |
+| '-----------------------------------'  Network Connections)   |
+|                                                               |
+|                       Composite Device                        |
+'---------------------------------------------------------------'
 ~~~~
 {: #composite title="Conceptual Data Flow for a Composite Device"}
 
 In the Composite Device, each Attester generates its own Evidence by its
 Attesting Environment(s) collecting the claims from its Target Environment(s).
-The Lead Attester collects the Evidence of all other Attesters and then
-generates the Evidence of the whole Composite Attester.
-
-The Lead Attester's Attesting Environment may or may not include its own
-Verifier.
-One situation is that the Attesting Environment has no local Verifier.
-In this situation, the Lead Attesting Environment simply combines the various
-Evidences into the final Evidence that is sent off to the remote Verifier,
-which evaluates the trustworthiness of the Composite Device,
-including the Lead Attester's and other Attesters', trustworthiness.
-
-The other situation is that the Lead Attesting Environment has an local Verifier.
-After collecting the Evidence of other Attesters, this Attesting Environment
-verifies them using Endorsements and Appraisal Policies (obtained the
-same way as any other Verifier), for evaluating these Attesters' trustworthiness.
-Then the Lead Attesting Environment combines the Attestation Results into
-the final Evidence of the whole Composite Attester which is sent off to the remote
-Verifier, which might treat the claims obtained from the local Attestation Results
-as if they were Evidence. In this situation, the trust model described
-in {{trustmodel}} is also suitable for the local verifier.
+The transiting Attester (e.g., Attester A) collects the Evidence of all other Attesters.
+It combines the Evidence from other Attesters with its own
```

Comment:
"(e.g., Evidence A from Attester A and Evidence B from Attester B)..."

Reply:
The Lead Attester collects the Evidence, is preferred.